crates/hyperqueue/src/client/commands/job.rs

```diff
@@ -36,19 +34,29 @@ pub struct JobListOpts {
 #[derive(Parser)]
 pub struct JobInfoOpts {
     /// Single ID, ID range or `last` to display the most recently submitted job
-    pub selector_arg: SelectorArg,
+    pub selector_arg: IdSelectorArg,
 }
 
 #[derive(Parser)]
 pub struct JobTasksOpts {
     /// Job ID
     pub job_id: u32,
+
+    /// Display only task(s) with the given states.
+    /// You can use multiple states separated by a comma.
+    #[clap(
+        long,
+        multiple_occurrences(false),
+        use_delimiter(true),
+        possible_values = &["waiting", "running", "finished", "failed", "canceled"]
+    )]
+    pub task_status: Vec<Status>,
```

Comment:
This help string is getting repeated a lot, it would be annoying/error-prone to change, especially the `possible_values` field. What about creating a struct that will contain both task ID and task status selector and then using it in `job tasks`, `job cat` and possibly other commands that allow filtering tasks? You can use `#[clap(flatten)] ` to "inline" the struct into its parent.